packages/@okta/vuepress-site/guides/sign-into-web-app/require-authentication/aspnet/reqautheverything.md

```diff
@@ -1,24 +1,24 @@
 To require authentication for all actions you can add the `Authorize` attribute in the filter config to apply it to every controller.
 
-```
+```csharp
 public class FilterConfig
 {
     public static void RegisterGlobalFilters(GlobalFilterCollection filters)
     {
-        ...
+        //...
         filters.Add(new AuthorizeAttribute());
     }
 }
 ```
 
-Also, you can opt to create a `BaseProtectedController`and make all of your controllers to inherit it:
+Or, you could create a `BaseProtectedController`and make all of your controllers to inherit from it:
```

Comment:
Or, you could create a `BaseProtectedController`and make all of your controllers inherit from it: